torch/testing/_internal/common_utils.py

```diff
@@ -1444,6 +1445,7 @@ def split_if_not_empty(x: str):
 
 NOTEST_CPU = "cpu" in split_if_not_empty(os.getenv('PYTORCH_TESTING_DEVICE_EXCEPT_FOR', ''))
 
+skipIfACL = unittest.skipIf(TEST_ACL, "ACL is not supported")
```

Comment:
Why not `xfailIfACL`

Reply:
I really like this suggestion 😸 my developer senses were tingling as I wrote it.